### ISSUE TITLE
Server is not up

### ISSUE THREAD

**Issue by PolarPeak** [PlatyPew/DNS-Reverse-Shell#2]
aliyun

Set an A record for *.evilhacker.com and point to server IP address
Set an NS record for ns1.evilhacker.com and point to ns1.evilhacker.com

运行client提示Server is not up 安全组已放行53端口

**Comment by PlatyPew** [comment]
The example stated assumes you have bought a domain `evilhacker.com`

To use this program, you must purchase your own domain, setup the necessary `A` and `NS` records and then use the program with your purchased domain

**Comment by PolarPeak** [comment]
Yes i bought testsec.com and set the A record to point to my ip address，set an NS record for ns1.testsec.com and point to ns1.testsec.com.
then I run ./dns_server.py -k test testsec.com

./dns_client.py -k test testsec.com

dns_client.py  show Server is not up

**Comment by PlatyPew** [comment]
Could you try using the `-p` flag for the server program and setting the public ip address manually?

**Comment by PolarPeak** [comment]
I have tried, set -p to point to my public network ip, and this ip is the ip pointed to by the A record, I should be fine doing this, but it still appears Server is not up

**Comment by PolarPeak** [comment]
Does the author have any successful screenshots or configuration screenshots? Can you refer to it?

**Comment by PlatyPew** [comment]
Hmm, you A record should be something like `*.testsec.com`

<img width="696" alt="image" src="https://user-images.githubusercontent.com/23520617/201481751-5d06393f-91fe-46e0-a724-fdedfa494066.png">

<img width="869" alt="image" src="https://user-images.githubusercontent.com/23520617/201481795-165be5b2-0b40-4c2a-8ebc-84ff5fa2e33c.png">

**Comment by PolarPeak** [comment]
did not receive return data，is there something wrong with my configuration?
<img width="743" alt="1668270466932" src="https://user-images.githubusercontent.com/44831329/201484353-efba6f9e-cbd5-456d-aa6b-ca29464792ad.png">

<img width="609" alt="1668270522865" src="https://user-images.githubusercontent.com/44831329/201484354-b2d0971f-3102-41e9-a016-c332a9a99205.png">

<img width="602" alt="1668270540351" src="https://user-images.githubusercontent.com/44831329/201484357-32e17d03-2d21-4ff2-924d-f29eabf690ec.png">
